src/Raven.Server/Documents/Indexes/IndexDefinitionBase.cs

```diff
@@ -135,10 +135,12 @@ public static class IndexVersion
 
             public const long Analyzers = 52_000;
 
+            public const long ReduceKeyProcessorHashDoubleFix = 52_001; // RavenDB-17572
```

Comment:
Please update the issue description so it will be accurate - https://issues.hibernatingrhinos.com/issue/RavenDB-17572